trin-storage/src/versioned/id_indexed_v1/store.rs

```diff
@@ -10,16 +10,16 @@ use crate::{
     error::ContentStoreError,
     utils::get_total_size_of_directory_in_bytes,
     versioned::{
-        usage_stats::{create_usage_stats_triggers, get_usage_stats, UsageStats},
-        ContentType, StoreVersion, VersionedContentStore,
+        sql::delete_usage_stats_triggers, usage_stats::UsageStats, ContentType, StoreVersion,
+        VersionedContentStore,
     },
     ContentId,
 };
 
-/// How frequently to check for pruning. Higher number means we will check more frequently while
-/// lower number (minimum 1) means we will check less frequently.
-const PRUNING_CHECKING_FREQUENCY: u64 = 100;
+/// The maximum number of entries to prune with a single query.
+const MAX_TO_PRUNE_PER_QUERY: u64 = 100;
```

Comment:
Same here w/ why this number was chosen & side effects to expect if it's changed down the road